examples/DotNetCore/ConsoleAppWithTableStorageReference/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
If we're keeping sample values here, can you add a comment explaining this is the content from Table storage? Also, we should add `MyShop:DisplayedColumns` here too.